queries.sql

```diff
@@ -42,4 +42,26 @@ SELECT COUNT(*) AS animal_never_escape FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) AS average_weight FROM animals;
 SELECT neutered, COUNT(*) AS escape_most FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth > '1990-01-01' AND date_of_birth < '2000-01-01' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth > '1990-01-01' AND date_of_birth < '2000-01-01' GROUP BY species;
+
+SELECT name FROM animals JOIN owners ON owners.id = animals.owner_id WHERE owners.full_name = 'Melody Pond';
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT owners.full_name AS owner_name, animals.name AS animal_name FROM owners LEFT JOIN animals ON animals.owner_id = owners.id;
```

Comment:
- [ ] The result for the query, `List all owners and their animals, remember to include those that don't own any animal.`  is incorrect. Your screenshot shows that Sam Smith has no animal assigned. Kindly fix the result of your query to only reflect `Jodie Whitaker` As the only Owner without an animal

Screenshot of requirements
![image](https://github.com/Esmatullah12/vet-clinic-database/assets/81932478/5d002fb5-c18d-4de2-b3d2-edc89960698e)

Screenshot of your query
![image](https://github.com/Esmatullah12/vet-clinic-database/assets/81932478/727975b4-b2e2-492d-ab0d-fb914fe40380)

Reply:
Hi @Munsa1 
The reason why Sam Smith doesn't have an animal named 'Augmon' is because of a project requirement on the second day. The requirement stated that all animals born after Jan 1st, 2022, should be deleted in a transaction. Additionally, a savepoint was created after the deletion, followed by multiplying the weight of the animals by -1. However, a rollback was performed to the savepoint, and then the weights of all animals with negative weights were updated to their weight multiplied by -1. Finally, the changes were committed. As a result, the 'Augmon' animal, which was born on Feb 3rd, 2020, was deleted. I have included a screenshot of the specific section from the second day project.
<img width="633" alt="Screenshot 2023-11-04 at 05 39 12" src="https://github.com/Esmatullah12/vet-clinic-database/assets/101590645/9dea2e4f-a377-46f9-a992-81774776fd91">

Reply:
Resolved on call